### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by afh**
as reported by [codespell](https://github.com/codespell-project/codespell).

### CODE REVIEW

File: src/plugins/cddadecoder/ntddcdrm.h [hunk 1]
```diff
@@ -503,7 +503,7 @@ typedef struct _CDROM_AUDIO_CONTROL {
 
 //
 // Volume control - Volume takes a value between 1 and 0xFF.
-// SCSI-II CDROM audio suppports up to 4 audio ports with
```
Comment (clangen) [inline src/plugins/cddadecoder/ntddcdrm.h]: This file is third-party, I'd prefer we preserve the typos in this case.

File: src/plugins/cddadecoder/ntddstor.h [hunk 2]
```diff
@@ -154,7 +154,7 @@ typedef struct _STORAGE_HOTPLUG_INFO {
 // output - STORAGE_DEVICE_NUMBER structure
 //          The values in the STORAGE_DEVICE_NUMBER structure are guaranteed
 //          to remain unchanged until the system is rebooted.  They are not
-//          guaranteed to be persistant across boots.
```
Comment (clangen) [inline src/plugins/cddadecoder/ntddstor.h]: This file is third-party, I'd prefer we preserve the typos in this case.

File: src/plugins/gmedecoder/gme/blargg_common.h [hunk 3]
```diff
@@ -107,7 +107,7 @@ we strip it out unless BLARGG_LEGACY is true. */
 
 /* BOOST::int8_t, BOOST::int32_t, etc.
 I used BOOST since I originally was going to allow use of the boost library
-for prividing the definitions. If I'm defining them, they must be scoped or
```
Comment (clangen) [inline src/plugins/gmedecoder/gme/blargg_common.h]: All files in the `src/plugins/gmedecoder/gme` directory are third-party -- I'd prefer we preserve the typos in this case.

File: src/plugins/libopenmptdecoder/libopenmpt/libopenmpt.h [hunk 4]
```diff
@@ -223,7 +223,7 @@ LIBOPENMPT_API void openmpt_free_string( const char * str );
  *        -  "source_revision": original source code revision
  *        -  "source_is_modified": "1" if the original source has been modified
  *        -  "source_has_mixed_revisions": "1" if the original source has been compiled from different various revision
- *        -  "source_is_package": "1" if the original source has been obtained from a source pacakge instead of source code version control
```
Comment (clangen) [inline src/plugins/libopenmptdecoder/libopenmpt/libopenmpt.h]: All files in the `src/plugins/libopenmptdecoder/libopenmpt` directory are third-party -- I'd prefer we preserve the typos in this case.

File: src/plugins/taglib_plugin/taglib-1.11/taglib/mpeg/id3v2/id3v2tag.h [hunk 5]
```diff
@@ -393,7 +393,7 @@ namespace TagLib {
       void setTextFrame(const ByteVector &id, const String &value);
 
       /*!
-       * Dowgrade frames from ID3v2.4 (used internally and by default) to ID3v2.3
```
Comment (clangen) [inline src/plugins/taglib_plugin/taglib-1.11/taglib/mpeg/id3v2/id3v2tag.h]: All files in the `src/plugins/taglib_plugin/taglib-1.11` directory are third-party -- I'd prefer we preserve the typos in this case.

**Review by clangen — COMMENTED** [review]
Small bit of feedback -- I'd prefer we preserve typos in third-party code; otherwise, looks good! :)

**Comment by afh** [comment]
Thanks for the feedback, @clangen, I went ahead and reverted the changes to the files in the directories you mentioned. It was not obvious to me that this was third-party code, that should be improved upstream.

**Comment by clangen** [comment]
Nice, thank you!

**Comment by afh** [comment]
Thanks for merging, @clangen 😃

**Pull request merged**